sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_delete_test_live.cpp

```diff
@@ -110,7 +102,6 @@ TEST_F(KeyVaultKeyClient, DeleteInvalidKey)
         static_cast<typename std::underlying_type<Azure::Core::Http::HttpStatusCode>::type>(
             error.StatusCode),
         404);
-    EXPECT_EQ(error.Message, GetNotFoundErrorMsg(keyName));
```

Comment:
Why did you remove this check?

Reply:
remeber we had this discussion regarding matching strings, the KV decided to update the message which would now cause the test to fail ( e.g. add a comma to the string )